modules/core/help.py

```
@@ -0,0 +1,295 @@
+            essential[key] = value
+        else:
+            module_[key] = value
+
+
+    if not show_disabled_modules:
+        module_ = {k: v for k, v in module_.items() if k in target_enabled_list}
+    
+    if show_base_modules:
+        module_list = {**essential, **module_}
+    else:
+        module_list = module_
```

Ruff

Use ternary operator `module_list = {**essential, **module_} if show_base_modules else module_` instead of `if`-`else`-block

Replace `if`-`else`-block with `module_list = {**essential, **module_} if show_base_modules else module_`

(SIM108)